lms/views/api/groups.py

```diff
@@ -1,15 +1,27 @@
+from marshmallow import Schema
 from pyramid.view import view_config
+from webargs import fields
 
 from lms.security import Permissions
+from lms.validation._base import PyramidRequestSchema
+
+
+class APIGroupSetsSchema(PyramidRequestSchema):
+    class LMS(Schema):
+        product = fields.Str(required=True)
```

Comment:
This was already being sent to allow request.product to correctly identify the right one.